examples/save_disp.py

```diff
@@ -1,67 +1,103 @@
 import os
+import sys
+from typing import List
 from pathlib import Path
 from packaging import version
+from argparse import ArgumentDefaultsHelpFormatter, ArgumentParser, Namespace
 
 import torch
 import numpy as np
 from PIL import Image
 
 
-def load_image(imfile):
-    """Utility function to load image files as tensors"""
-    img = Image.open(imfile)
+def load_image(im_path: Path, device: str = "cpu") -> torch.Tensor:
+    """Utility to load an image file as a torch tensor.
+
+    Args:
+        im_path (pathlib.Path): The path to the image file on the system.
+        device (str)    : The device on which the tensor should be loaded.
+
+    Returns:
+        A torch tensor of the image on the specified device.
+    """
+
+    img = Image.open(im_path)
     img = np.array(img).astype(np.uint8)
+    assert img.ndim == 3, f"Expected 3D tensor, got shape {img.shape}"
+
     img = torch.from_numpy(img).permute(2, 0, 1).float()
-    img = img.to("cuda")
+    img = img.to(device)
 
-    assert img.ndim == 3, f"Expected 3D tensor, got shape {img.shape}"
     return img
 
 
-def main():
+def parse_args(arg_list: List[str] = sys.argv[1:]) -> Namespace:
+    """ Parse the command line arguments.
+
+	Args:
+		arg_list: The list of arguments to be parsed.
+
+	Returns:
+		The parsed command line arguments.
+	"""
+
+    parser = ArgumentParser(
+        description='Script for exporting MonSter to a torchscript module.',
+        formatter_class=ArgumentDefaultsHelpFormatter,
+		allow_abbrev=False
+    )
+
+    parser.add_argument("model",            help="Path to the torchscript model", type=Path)
+    parser.add_argument("left_image",       help="Path to the left image in the stereo pair", type=Path)
+    parser.add_argument("right_image",      help="Path to the right image in the stereo pair", type=Path)
+    parser.add_argument("disparity_path",   help="Path to save the disparity image to", type=Path)
+
+    return parser.parse_args(arg_list)
+
+
+def main() -> None:
+    args = parse_args()
+
+    assert os.path.exists(args.model), f"Given model file does not exist: {args.model}"
```

Comment:
Same for the others.
```suggestion
    assert args.model.is_file(), f"Given model file does not exist: {args.model}"
```